spring-sample-app/pom.xml

```diff
@@ -5,29 +5,20 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.1.7.RELEASE</version>
-		<relativePath/> <!-- lookup parent from repository -->
+		<version>2.2.6.RELEASE</version>
+		<relativePath/>
 	</parent>
-	<groupId>com.cosmin.tutorials</groupId>
-	<artifactId>spring-reactive</artifactId>
+	<groupId>io.github.cosminseceleanu.tutorials.sampleapp</groupId>
+	<artifactId>spring-sample-app</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<name>spring-reactive</name>
-	<description>Demo project for Spring Boot</description>
+	<name>spring-sample-app</name>
+	<description>A Spring Boot aplication used by various tutorials</description>
```

Comment:
**[misspell]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>"aplication" is a misspelling of "application"